app/models/prison.rb

```diff
@@ -49,11 +49,16 @@ def confirm_by(today = Time.zone.today)
   end
 
   def validate_visitor_ages_on(target, field, ages)
+    return if ages.blank?
```

Comment:
@alan do you understand the reason for this return? It seems non-obvious. I think we should find a way to avoid this, or at least add a comment.